wasm2c/README.md

```diff
@@ -143,34 +143,52 @@ You can take a look at the all of these files in
 The generated header file looks something like this:
 
 ```c
+/* Automatically generated by wasm2c */
 #ifndef FAC_H_GENERATED_
 #define FAC_H_GENERATED_
 
-...
+#include <stdint.h>
 
 #include "wasm-rt.h"
 
-...
+#if defined(WASM_RT_ENABLE_SIMD)
+#include "simde/wasm/simd128.h"
+#endif
+
+/* TODO(binji): only use stdint.h types in header */
 #ifndef WASM_RT_CORE_TYPES_DEFINED
 #define WASM_RT_CORE_TYPES_DEFINED
-
-...
+typedef uint8_t u8;
+typedef int8_t s8;
+typedef uint16_t u16;
+typedef int16_t s16;
+typedef uint32_t u32;
+typedef int32_t s32;
+typedef uint64_t u64;
+typedef int64_t s64;
+typedef float f32;
+typedef double f64;
+
+#if defined(WASM_RT_ENABLE_SIMD)
+typedef simde_v128_t v128;
+#endif
```

Comment:
Why expand the `...` here?

Reply:
Nice catch... I had just pasted in the current code and didn't match the `...` from before. Fixed in bdcd93dffdbd1138b79714978cf783c88078956c